search-service/src/test/resources/ngsild/entities_with_two_instances_who_have_same_dataset_id.jsonld

```diff
@@ -0,0 +1,857 @@
+{
```

Comment:
do you **really** need such a laaarge payload?

Reply:
No, I can just leave what we need